nni/retiarii/converter/graph_gen.py

```diff
@@ -249,6 +251,15 @@ def _generate_expr(tensor):
                     return f'({left} < {right})'
                 elif tensor.node().kind() == 'prim::If':
                     raise RuntimeError('Have not supported `if A and/or B`, please use two `if` statements instead.')
+                elif tensor.node().kind() == 'aten::abs':
```

Comment:
Suggest add tests for newly added operators and python name.

Reply:
Got it. Will fix it soon.